src/main.js

```diff
@@ -1,14 +1,29 @@
 // Globals
-window.worldData
 window.game = new Phaser.Game(1000, 600, Phaser.Auto, '');
 
-startGame();
+var Main = function() {};
+var worldData;
```

Comment:
I think this will need to be set as `window.worldData` on order to a global variable, because it's going to need to be accessible across all states. But I agree it doesn't really belong in window, perhaps an add on to `game`? So `game.worldData`.